.gitlab-ci.yml

```diff
@@ -281,43 +282,56 @@ topPRMEMC-sim:
   variables:
     VIVADO_VERSION: "2019.2"
     PROJ_NAME: "PRMEMC"
-  needs: # needed to avoid interference from ME-vitis-hls-build
+  needs:
+    - download
+    - PR-quality-check
+    - ME-quality-check
+    - MC-quality-check
```

Comment:
Don't you want the `{PR, ME, MC}-vivado-hls-build` jobs to complete first? Why reduce the requirement just to the quality check jobs?